iree/tools/run_mlir_main.cc

```diff
@@ -267,11 +289,20 @@ Status EvaluateFunction(iree_vm_context_t* context,
 
   std::cout << "EXEC @" << export_name << std::endl;
   ASSIGN_OR_RETURN(auto input_descs, ParseInputSignature(function));
-  auto input_values_list = absl::MakeConstSpan(
-      input_values_flag.empty() ? nullptr : &input_values_flag.front(),
-      input_values_flag.size());
-  ASSIGN_OR_RETURN(auto* input_list, ParseToVariantList(input_descs, allocator,
-                                                        input_values_list));
+  iree_vm_variant_list_t* input_list;
+  if (!input_values_file_flag.empty()) {
```

Comment:
Validation that only one of input_values_file_flag and input_values_flag is set?

Reply:
Done